etc/env/example.env

```diff
@@ -130,9 +130,8 @@ TOIL_WORKER_PORT=5051
 FEDERATION_VERSION=v2.1.1
 FEDERATION_IP=0.0.0.0
 FEDERATION_PORT=4232
-FEDERATION_SERVICE_URL=http://${CANDIG_INTERNAL_DOMAIN}:${FEDERATION_PORT}
+FEDERATION_SERVICE_URL=http://federation:${FEDERATION_PORT}
```

Comment:
This is supposed to be different than FEDERATION_PRIVATE_URL: in a context like federation_setup.sh, which is not called inside the container, the URL needs to be an internal one, not the container URL.